### ISSUE TITLE
testCanBeEncoded fails

### ISSUE THREAD

**Issue by remicollet** [zendframework/zend-mail#183]
From Fedora QA
https://apps.fedoraproject.org/koschei/package/php-zendframework-zend-mail

```
There was 1 failure:
1) ZendTest\Mail\Header\HeaderWrapTest::testCanBeEncoded
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 'Subject: =?UTF-8?Q?[#77675]=20New=20Issue:xxxxxxxxx=20xxxxxxx=20xxxxxxxx=20?=\r\n
- =?UTF-8?Q?xxxxxxxxxxxxx=20xxxxxxxxxx=20xxxxxxxx,=20t=C3=A4htaeg=20xx.xx,=20xxxx?='
+ =?UTF-8?Q?xxxxxxxxxxxxx=20xxxxxxxxxx=20xxxxxxxx=2C=20t=C3=A4htaeg=20?=\r\n
+ =?UTF-8?Q?xx.xx=2C=20xxxx?='
/builddir/build/BUILD/zend-mail-248230940ab1453b2a532a8fde76c5a6470d7aad/test/Header/HeaderWrapTest.php:97
```

Seems to be related to **zend-mime update to 2.7.0** (first failed build, Dec 1th)

**Comment by glensc** [comment]
refs: https://github.com/zendframework/zend-mime/pull/26

**Comment by Ocramius** [comment]
Fixed by #189
